### PR TITLE
Add docs for IDE0260, IDE0270

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #30361 

---

#### Internal previews

<details><summary><strong>Toggle expand/collapse</strong></summary><br/>

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/csharp/language-reference/operators/member-access-operators.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/csharp/language-reference/operators/member-access-operators.md) | [Member access operators and expressions - the dot, indexer, and invocation operators.](https://review.learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/member-access-operators?branch=pr-en-us-36394) |
| [docs/csharp/language-reference/operators/null-coalescing-operator.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/csharp/language-reference/operators/null-coalescing-operator.md) | ["?? and ??= operators - null-coalescing operators"](https://review.learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/ide0019.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/ide0019.md) | [Use pattern matching to avoid 'as' followed by a 'null' check (IDE0019)](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0019?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/ide0029-ide0030-ide0270.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/ide0029-ide0030-ide0270.md) | [docs/fundamentals/code-analysis/style-rules/ide0029-ide0030-ide0270](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0029-ide0030-ide0270?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/ide0078-ide0260.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/ide0078-ide0260.md) | [docs/fundamentals/code-analysis/style-rules/ide0078-ide0260](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0078-ide0260?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/ide0083.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/ide0083.md) | [docs/fundamentals/code-analysis/style-rules/ide0083](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0083?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/ide0084.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/ide0084.md) | [Use pattern matching (`IsNot` operator) (IDE0084)](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0084?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/ide0140.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/ide0140.md) | ["IDE0140: Simplify object creation"](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0140?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/ide0200.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/ide0200.md) | [Remove unnecessary lambda expression (IDE0200)](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0200?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/ide0210.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/ide0210.md) | [docs/fundamentals/code-analysis/style-rules/ide0210](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0210?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/ide0211.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/ide0211.md) | [Convert to 'Program.Main' style program (IDE0211)](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0211?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/ide0220.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/ide0220.md) | ["IDE0220: Add explicit cast in foreach loop"](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0220?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/ide0230.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/ide0230.md) | [Use UTF-8 string literal (IDE0230)](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0230?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/ide0250.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/ide0250.md) | [docs/fundamentals/code-analysis/style-rules/ide0250](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0250?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/ide0251.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/ide0251.md) | [Member can be made 'readonly' (IDE0251)](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0251?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/index.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/index.md) | [Code-style rules overview](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/index?branch=pr-en-us-36394) |
| [docs/fundamentals/code-analysis/style-rules/language-rules.md](https://github.com/dotnet/docs/blob/a4e7f34254937ce2c248711aa32cabe1003dbcc4/docs/fundamentals/code-analysis/style-rules/language-rules.md) | [Language and unnecessary rules](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/language-rules?branch=pr-en-us-36394) |

</details>

### CODE REVIEW

File: docs/fundamentals/code-analysis/style-rules/ide0029-ide0030.md
Comment (Youssef1313) [inline docs/fundamentals/code-analysis/style-rules/ide0029-ide0030.md:1]: Is this file missing a rename to include ide0270?

**Review by BillWagner — APPROVED** [review]
This LGTM @gewarren Let's :shipit:

**Comment by gewarren** [comment]
Thanks for reviewing @BillWagner.

File: docs/fundamentals/code-analysis/style-rules/ide0078-ide0260.md [hunk 1]
```diff
@@ -5,16 +5,18 @@ ms.date: 09/30/2020
 ms.topic: reference
 f1_keywords:
 - IDE0078
+- IDE0260
 - csharp_style_prefer_pattern_matching
 helpviewer_keywords:
 - IDE0078
+- IDE0260
 - csharp_style_prefer_pattern_matching
```
Comment (Youssef1313) [inline docs/fundamentals/code-analysis/style-rules/ide0078-ide0260.md:13]: Is `csharp_style_pattern_matching_over_as_with_null_check` missing in f1_keywords and helpviewer_keywords?

**Pull request merged**